kustomize/overlays/sandbox/kustomization.yaml

```diff
@@ -38,16 +36,13 @@ configMapGenerator:
   - behavior: merge
     files:
       - ./config/clusterresource-templates/ac_project-copilot-dataconfig.yaml
-      - ./config/clusterresource-templates/ad_spark-role.yaml
-      - ./config/clusterresource-templates/ae_spark-service-account.yaml
-      - ./config/clusterresource-templates/af_spark-role-binding.yaml
     name: clusterresource-template
   - behavior: merge
     files:
       - ./config/propeller/enabled_plugins.yaml
+      - ./config/propeller/resource_manager.yaml
       - ./config/propeller/plugins/k8s.yaml
       - ./config/propeller/plugins/qubole.yaml
```

Comment:
Can this be dropped too?

Reply:
This file is what defines the resource manager as "Noop"... let me rename it to make it clearer

Reply:
Sorry I was referring to qubole. I guess that didn’t get represented correctly!